### PR TITLE
Reworks the Twinkleshine-Class

### PR DESCRIPTION
**Pull request by meemofcourse**
## About The Pull Request

twinklerework5

The Twinkleshine is a CyberSun-made Syndicate display of force, staffed by every branch of the Syndicate. Despite the silly name, the presence of one in a sector implies it to be of importance to the Syndicate, and enemies within sight can only pray that the Twinkleshine crew are incompetent enough to shoot themselves with their own weaponry (or blow themselves up with the supermatter on-board).

It is staffed by:

- 1 Captain
- 1 Lieutenant (previously the Operative - serves as a warden/hos)
- 2 Medics
- 2 Engineers (previously the Mechanics)
- 5 Operatives (previously the Troopers)
- 1 Bartender
- 1 Miner
- 2 Deck Assistants

## Why It's Good For The Game

Few days ago, an admin spawned a Twinkleshine, and I got to captain it. The Twinkleshine is old. It sucks. This, hopefully, fixes that.

Originally, this was going to be minor fixes, but ended up becoming an attempt at reworking the ship to a more modern state - the hull has been redone and is mostly symmetrical, the old spacepod was replaced with a Dark Gygax, the supermatter shouldn't be activated upon spawning the ship, there's more turf decals and a bigger lot-of-things, added a bartender and a miner, people can actually tell who they are and what they do, and there is now a box moth. Rejoice.

Also, this is the first time I've ever mapped a ship. What better way to begin with a giant battleship?

## Changelog

:cl:
tweak: Reworks the Twinkleshine
/:cl:

### CODE REVIEW

**Review by rye-rice — APPROVED** [review]
thank you so much for keeping the twinkleshine up to par

**Comment by meemofcourse** [comment]
mam.p

**Comment by Zevotech** [comment]
image

**Comment by meemofcourse** [comment]
wall moth! wall moth! wall moth!

**Review by spockye — CHANGES_REQUESTED** [review]
image
vendor is a bit akwardly placed. perhaps push forward a bit? 

image
one bed blocks cryo, another is trapped into the corner. Move the one infront of cryo to the side. Do also give them some more normal backpacks and headsets. (also wall light)

image
The cargo bay could use a rework.  The holofans also aren't connected to power


image
try to open this up a bit. it's way too compact
im also not a huge fan of how it looks. The decalling is a bit too repeating, and it could use some more touches

image
Not a fan of them having a mech, seeing the already insane amount of guns they have in the armoury

image
missing wall. 

image
Piping errors

image
Cable errors. Dorms and cargo lack an APC

Captains locker has two 367 loaders, but no gun. 

the telecomms computer in comms is pixelshifted, putting it under the other computer

i'd appreciate if some of the plasma enginers were replaced with ion

the buttons for the launch bays seem switched around?


image
locker in each corner, blocking you from going around without opening them. why is there a coffee vendor in the armoury? Wardens locker is kinda weirdly placed, leaving a empty corner.
im not a fan of the armoury. it feels cramped, while still having a lot of open spaces, and a basically complete lack of decalling. Give the windoors access "1" so people without id's cant open it. the wardens office also needs access restrictions

---
All in all its definitely an upgrade, but i do think theres a lot that still needs changing.
Good work!

**Comment by meemofcourse** [comment]
ah, so i _do_ suck at mapping
> Captains locker has two 367 loaders, but no gun.

the captain starts with a gun by default. so do the assistants (i'll change that later) and the lieutenant
> Not a fan of them having a mech, seeing the already insane amount of guns they have in the armoury

i'll admit, having them start with a mauler is overkill, but this was meant as a replacement for the space pod they had in the hangar. if you have any suggestions as to what it could be replaced with (or if i shouldn't bother) do tell

**Comment by Ms-Mee** [comment]
A Ripley painted black WOULD be kinda funny

**Comment by meemofcourse** [comment]
that does sound funny

**Comment by meemofcourse** [comment]
imagen
literally 1984

**Comment by meemofcourse** [comment]
clicky button

**Comment by Apogee-dev** [comment]
You know, if a Mauler is overkill (and it is) and a Ripley is underwhelming (which it is) you could give them a Dark Gygax instead. Still a combat mech but not completely ridiculous.

**Comment by meemofcourse** [comment]
that sounds like a good in-between

**Comment by pitaden** [comment]
If your initial clothes get destroyed, the ship doesn't really have spare clothes/shoes for you to grab

**Comment by Erikafox** [comment]
@spockye remember to lift / approve once your changes are addressed

**Review by Erikafox — CHANGES_REQUESTED** [review]
image
You need at least 2 high volume dual port air vents for an airlock of this size.
I would also /consider/ adding an interior turret. It is a battlecruiser after all.
StrongDMM_sO2u0njUpM
these won't layer over the catwalks, so it'll end up looking like shit. 
image
bad spot for these lockers. 
StrongDMM_kum9b71beg
The doublethick decalling won't look /great/, I'd remove the interior line (and the one running on the circuit tiles too)
StrongDMM_mQ9ZwqHJpB
same decal layering thing here
StrongDMM_AlBVX8nyXL
wall moth is hilarious, but will cause the walls to render weirdly. It should go.
StrongDMM_TaKClmY89V
Give the mech mech ions or a mech jetpack, otherwise it's going to suck too much ass in space.
StrongDMM_XmtvkkXmiG
move the blast door/pike back by 1 and consider adding some railings/greebling (look at the heron pr's mech launcher) to the spikes.


There's no captain's room like there was before, and I have a few interior design choices I would like to go over with you. Ping me in the discord when you're ready

**Comment by meemofcourse** [comment]
fuck

**Comment by meemofcourse** [comment]
cooltext432286312848364

**Comment by Erikafox** [comment]
the next review is watching it get played either friday or saturday.

**Review by spockye — CHANGES_REQUESTED** [review]
image
cable errors

image
still haven't fixed this

image
more cable errors

image
please switch the tiles outside with the airless subtype

image
piping errors

image
piping errors

image
piping error

image
no need for wall pipes here!

**Comment by meemofcourse** [comment]
d

**Comment by goober3** [comment]
image
bad prechargers!

image
u should make the floor inside of the sidings different. for Texture. in my opinion

image
replace the atmos tech locker with an engineer locker. and put an RPD in both of them. both engineers need RPDs for engineering tomfoolery. also consider: Engineering Hardsuits 

image
image
maybe find a way to get rid of these Doors. they break up the structure of the ship a little. absolutely not your fault but the engineering department probably shouldn't have access to the armory. especially now that the ship has AACs on the front, too

I think it would be better for this ship to use normal IDs. operative IDs are nice but having a bank account is nicer.

**Comment by AverageUser67** [comment]
ADD A FUCKING AUTOLATHE

**Comment by meemofcourse** [comment]
SHUT THE HELL UP I DID THAT ALREADY!!!!!!

**Comment by BogCreature** [comment]
The engineering and captain IDs don't have access to their respective departments, meaning you can't download apps with them via the app downloading thingy

File: _maps/shuttles/shiptest/syndicate_twinkleshine.dmm
Comment (Zevotech) [inline _maps/shuttles/shiptest/syndicate_twinkleshine.dmm]: REMOVE THIS! Spawnwithmagazine removes the magazine of the weapon itself instead of the ammo, which results in the PML 9 spawning without an internal magazine and being unable to be loaded.

**Comment by meemofcourse** [comment]
god

**Comment by meemofcourse** [comment]
> The engineering and captain IDs don't have access to their respective departments, meaning you can't download apps with them via the app downloading thingy

imagen
i think you just suck. the engi id can't download CIMS though you're right

**Comment by meemofcourse** [comment]
> I think it would be better for this ship to use normal IDs. operative IDs are nice but having a bank account is nicer.

you have an account id in your notes you can just link it if you alt click your id card. i'll try to automate it maybe

**Comment by Erikafox** [comment]
image
Consider swapping the position of the hall and the quarters here. It should avoid radiation poisoning, because I keep seeing the rooms starting to get green.

**Comment by meemofcourse** [comment]
imagen
ok there you go

**Comment by Erikafox** [comment]
image
Hull shutter

**Comment by Erikafox** [comment]
dreamseeker_JPlrOkqaSp
this is in engineering for some reason

**Comment by Erikafox** [comment]
dreamseeker_UzhBlqaVrR
this corner in med is a smidge dark

**Comment by meemofcourse** [comment]
> dreamseeker_JPlrOkqaSp this is in engineering for some reason

noted. messed it up with fixing a merge conflict

**Comment by goober3** [comment]
image
this Turret is still shooting at the engines 

image
you could swap this console and the operating table around so doctors can perform surgery on two people at once

engineers still get inducers in their belt, this is bad!
an outpost console would be nice

**Comment by goober3** [comment]
oh, and holofans are needed

**Comment by Erikafox** [comment]
image
upper button doesnt work

**Comment by goober3** [comment]
> image upper button doesnt work

I think there are quite a few buttons on this ship that don't work. worth checking over all of them

**Comment by meemofcourse** [comment]
probably!

**Comment by Valorium-RB** [comment]
> imagen
> ok there you go

Lizard wine...?

**Comment by meemofcourse** [comment]
> Lizard wine...?

actually that's genuine moth wine, made from genuine moth wings, harvested from genuine box twink moths. do you have a problem with it?

**Review by spockye — DISMISSED** [review]
image

i dislike the plants clipping into the lights. please move the plants, or lights

image

missing pipes

image

missing wires

image

accidental floor?

image

move the light

image

move the light (try to avoid having lights in front of wallmounts)

image

try to keep yer wall lockers on +-28

image

this crate blocks the table

image

dont have to, but you could probably just make this a subtype

**Review by spockye — COMMENTED** [review]
image
(i do still think the mech could be ditched)

**Comment by meemofcourse** [comment]
> (i do still think the mech could be ditched)

then what do i replace the room with? :(

**Comment by spockye** [comment]
> then what do i replace the room with? :(

same as on the other side, perhaps broken?

**Comment by meemofcourse** [comment]
right, but what would I replace the mech with? seeing as how the spacepod the original housed is gone, they still need a funky Battlecruiser weapon

**Comment by Zevotech** [comment]
> right, but what would I replace the mech with? seeing as how the spacepod the original housed is gone, they still need a funky Battlecruiser weapon

give them ATV's. SYNDICATE ATV'S.

**Comment by meemofcourse** [comment]
> > right, but what would I replace the mech with? seeing as how the spacepod the original housed is gone, they still need a funky Battlecruiser weapon
> 
> give them ATV's. SYNDICATE ATV'S.

ok so apparently there's an atv subtype that spawns a turret and puts it on the atv. it does not work when spawned via a ship though so no dice

**Review by Erikafox — APPROVED** [review]

**Comment by Erikafox** [comment]
Leave the mech it's an admin ship. Good setpiece.

**Review by spockye — COMMENTED** [review]
image

missing wire (engines)

image

pipe fuckery?

----


Looks good!

**Comment by meemofcourse** [comment]
> pipe fuckery?

intentional, leaving it up to engineers to their own devices with it since the sm doesn't produce nitrogen

**Comment by meemofcourse** [comment]
also, added the mech back

**Review by Erikafox — CHANGES_REQUESTED** [review]
This needs at least 1 external stationary docking port. Ping me in ruinyard or look at the Borea if you need an example

**Comment by meemofcourse** [comment]
damn pirates

**Comment by Erikafox** [comment]
I was gonna merge this but your int checks are failing.

**Comment by meemofcourse** [comment]
oh, that's because i moved doctor to brig phys. i would recommend testing it at least once before merging however since i changed a moderate amount of stuff

**Comment by Erikafox** [comment]
sure we can do that wednesday but also fix your integration checks

**Comment by meemofcourse** [comment]
merge my fucking pr

**Comment by Erikafox** [comment]
yeah sure

**Comment by meemofcourse** [comment]
@spockye @Bjarl close ur reviews

**Review by Erikafox — APPROVED** [review]

**Pull request merged**
